snmp/datadog_checks/snmp/data/default_profiles/ubiquiti-unifi.yaml

```diff
@@ -0,0 +1,71 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+
+sysobjectid:
+  - 1.3.6.1.4.1.41112  # Ubiquiti Networks, Inc.
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "ubiquiti"
```

Comment:
I see Kentik has several profiles of this vendor, if we are planning to add them, it's probably better to create `ubiquity.yaml` now and to extend it. What do you think ?

Reply:
Good idea 👍